prime-router/src/main/kotlin/SettingsProvider.kt

```diff
@@ -53,6 +54,7 @@ enum class Topic(
     val isUniversalPipeline: Boolean = true,
     val isSendOriginal: Boolean = false,
     val validator: IItemValidator = NoopItemValidator(),
+    val hl7ParseConfiguration: HL7Reader.Companion.HL7MessageParseAndConvertConfiguration? = null,
```

Comment:
Why wasn't MARS_OTC_ELR updated to configure `HL7MessageParseAndConvertConfiguration`?

Reply:
I was holding off on adding that until the mappings and java file is done.

Reply:
FYI, my thought is that the fast follow is done in [this](https://github.com/CDCgov/prime-reportstream/pull/14255) PR, let me know if you have a different preference